doc/nrf/gs_assistant.rst

```diff
@@ -9,11 +9,8 @@ Installing automatically
 
 `nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS, provides different apps that simplify installing the |NCS|.
 
-.. note::
-   When installing the |NCS| through the |GSA| app, you will not be able to build on the command line using west.
-
 On Windows or macOS, use the :ref:`gs_app_tcm`.
-On Linux, use the :ref:`gs_app_gsa` or follow the steps found on the :ref:`gs_installing` page.
+On Linux, use the `Installing using Visual Studio Code <Installing on Linux_>`_ or follow the steps found on the :ref:`gs_installing` page.
```

Comment:
Having "the" before the link currently sounds a little strange. I would suggest the following:

```suggestion
On Linux, use the `Installing using Visual Studio Code <Installing on Linux_>`_ instructions or follow the steps found on the :ref:`gs_installing` page.
```